### ISSUE TITLE
Using CenterNet with Torch 1.x

### ISSUE THREAD

**Issue by deandiakov** [xingyizhou/CenterNet#446]
Has anyone had any experience / success porting CenterNet to work with PyTorch 1.0 or higher? From the attempts I've done there seems to be errors with a missing Nvidia library link every time. Please let me know!

**Comment by mathpopo** [comment]
And I use https://github.com/CharlesShang/DCNv2.git replace your code
pytorch1.2 cuda10.0
is ok!!!!!!!!!!!!!!!!!!!

**Comment by LaCandela** [comment]
@mathpopo HI! Did you just simply replace the CenterNet/src/lib/models/networks/DCNv2 directory with the pytorch1.0 version of DCNv2 taht you linked above? Does it generate the models.networks.DCNv2._ext module as well which is reguired to run for example the demo.py script?

**Comment by mathpopo** [comment]
@LaCandela    yes, i just mv the old DCNv2 file folder,replace the new git code ,left else operations also as same as before.

**Comment by edbean98** [comment]
> @LaCandela yes, i just mv the old DCNv2 file folder,replace the new git code ,left else operations also as same as before.

他说编译错误是怎么回事？
你有编译好的文件吗？

**Comment by gtfaiwxm** [comment]
@mathpopo After I simply replace the CenterNet/src/lib/models/networks/DCNv2 directory, the command line returns the Segmentation fault (core dumped) error. Could you help me?

**Comment by LaCandela** [comment]
@mathpopo thanks, it works fine!

**Comment by kp-1** [comment]
excuse me,dose anyone have this issue:ModuleNotFoundError: No module named '_ext'?

**Comment by LaCandela** [comment]
@kp-1 Did you install the deformable convolution repo?

**Comment by FranciscoReveriano** [comment]
Yeah i just got that issue.

**Comment by sohail759** [comment]
RESOLVED 
https://www.gitmemory.com/issue/xingyizhou/CenterNet/7/486653333
